babel/core.py

```diff
@@ -885,6 +885,41 @@ def ordinal_form(self):
         """
         return self._data.get('ordinal_form', _default_plural_rule)
 
+    @property
+    def measurement_systems(self):
+        """Localized names for various measurement systems.
+
+        >>> Locale('fr', 'FR').measurement_systems['US']
+        u'am\\xe9ricain'
+        >>> Locale('en', 'US').measurement_systems['US']
+        u'US'
+
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
+        """The text direction for the language.
+
+        >>> Locale('de', 'DE').character_order
+        'left-to-right'
+        >>> Locale('ar', 'SA').character_order
+        'right-to-left'
+        """
+        return self._data['character_order']
+
+    @property
```

Comment:
Yikes, good catch. :) What do you think about transforming "left-to-right" to "ltr" (yah, I'm still trying to get rid of one of the two methods :)